### PR TITLE
LPS 145980 Move React components to compotents folder

### PR DESCRIPTION
**Pull request by claraizquierdo**
Motivation
=======
We want to improve the organization of the javascript code
[Link to story or ticket](https://issues.liferay.com/browse/LPS-145980)

Proposed Solution
========
We have moved all the React components to the components folder. Also grouped all components used only in AuditGraphApp to the AuditGraphApp folder.

Steps to Verify:
----------------
1. Go to the Content Dashboard and check that everything works fine

Change summary (optional):
---------------
* LPS-145980 Move content dashboard components to the components folder
* LPS-145980 Move AuditGraphApp component to AuditGraphApp subfolder

### CODE REVIEW

**Comment by claraizquierdo** [comment]
ci:test:sf

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SelectTypeAndSubtype.js [hunk 1]
```diff
@@ -15,8 +15,8 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 
-import TreeFilter from './components/TreeFilter';
-import {nodeTreeArrayMapper} from './utils/tree-utils';
+import {nodeTreeArrayMapper} from '../utils/tree-utils';
```
Comment (beltranrengifo) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SelectTypeAndSubtype.js:18]: @claraizquierdo can you please:
- move the tree utils to `/components/TreeFilter/utils`?
- rename the file to camelcase `treeUtils.js`, as it should be when I created it?
I wanted to do this for a while and I always forgot :(
Txs!

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/edit_content_dashboard_configuration.jsp [hunk 2]
```diff
@@ -59,7 +59,7 @@ ContentDashboardAdminConfigurationDisplayContext contentDashboardAdminConfigurat
 						</span>
 
 						<react:component
-							module="js/VocabulariesSelectionBox"
+							module="js/components/VocabulariesSelectionBox"
```
Comment (beltranrengifo) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/edit_content_dashboard_configuration.jsp:62]: I wonder if `VocabulariesSelectionBox.js` should be inside `AuditGraphApp` as it is coupled and serves the chart 🤔

**Review by beltranrengifo — COMMENTED** [review]
Hi @claraizquierdo ! Just a couple of thoughts here...

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SelectTypeAndSubtype.js (hunk 1 above)
Comment (claraizquierdo) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SelectTypeAndSubtype.js:18]: Ok Beltrán, I agree with you :)

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/edit_content_dashboard_configuration.jsp (hunk 2 above)
Comment (claraizquierdo) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/edit_content_dashboard_configuration.jsp:62]: Mmmm, it is true that it is related to the graph, but it is also related to the content table. I would leave it under the components folder.

**Review by beltranrengifo — APPROVED** [review]
🎉

**Comment by claraizquierdo** [comment]
ci:test:relevant

**Comment by claraizquierdo** [comment]
ci:test:tango

**Comment by claraizquierdo** [comment]
ci:forward

**Comment by claraizquierdo** [comment]
ci:reevaluate:1297401_1463

**Comment by claraizquierdo** [comment]
ci:test:relevant

**Comment by beltranrengifo** [comment]
ci:reevaluate:1277401_1563

**Comment by claraizquierdo** [comment]
ci:forward

**Pull request closed without merging**